zoomage_report.CURATED.tsv

```diff
@@ -55965,3 +55965,225 @@ cell type	normal human epidermal keratinocyte	http://purl.obolibrary.org/obo/CL_
+compound	nickel (II) chloride hexahydrate	http://purl.obolibrary.org/obo/CHEBI_53542	E-GEOD-6907	HepG2_Ni rep3	Anja Fullgrabe	13/01/21 18:40
+compound	17 beta-estradiol	http://purl.obolibrary.org/obo/CHEBI_16469	E-GEOD-57804	SRR1291672	Anja Fullgrabe	13/01/21 18:40
+compound	17-beta estradiol	http://purl.obolibrary.org/obo/CHEBI_16469	E-GEOD-46890	GSM1139870	Anja Fullgrabe	13/01/21 18:40
+compound	abscisic acids	http://purl.obolibrary.org/obo/CHEBI_22152	E-GEOD-114379	SRR7160925	Anja Fullgrabe	13/01/21 18:40
+compound	abscisic acids	http://purl.obolibrary.org/obo/CHEBI_22152	E-GEOD-99677	SRR5645188	Anja Fullgrabe	13/01/21 18:40
+compound	azetidine-2-carboxylic acid	http://purl.obolibrary.org/obo/CHEBI_38108	E-GEOD-100371	SRR5742213	Anja Fullgrabe	13/01/21 18:40
+compound	azetidine-2-carboxylic acid	http://purl.obolibrary.org/obo/CHEBI_38108	E-MTAB-9355	ERR4344161	Anja Fullgrabe	13/01/21 18:40
+compound	etoposide	http://purl.obolibrary.org/obo/CHEBI_4911	E-MTAB-9704	ERR4768738	Anja Fullgrabe	13/01/21 18:40
+compound	gamma-secretase inhibitor DAPT	http://purl.obolibrary.org/obo/CHEBI_86193	E-GEOD-6495	MOLT4_GSI_C	Anja Fullgrabe	13/01/21 18:40
+compound	insulin-like growth factor-1	http://purl.obolibrary.org/obo/CHEBI_80343	E-MTAB-2951	IGF1	Anja Fullgrabe	13/01/21 18:40
+compound	(-)-methyl jasmonate	http://purl.obolibrary.org/obo/CHEBI_15929	E-MTAB-5413	ERR1806561	Anja Fullgrabe	13/01/21 18:40
+compound	trimethylamine N-oxide	http://purl.obolibrary.org/obo/CHEBI_15724	E-MTAB-9869	ERR4970155	Anja Fullgrabe	13/01/21 18:40
+compound	vitamin D3	http://purl.obolibrary.org/obo/CHEBI_73558	E-MTAB-9789	ERR4859272	Anja Fullgrabe	13/01/21 18:40
+cultivar	Williams82	http://www.ebi.ac.uk/efo/EFO_0007023	E-MTAB-5066	SRR1657450	Anja Fullgrabe	13/01/21 18:40
+developmental stage	FL.00 first flower(s) open stage	http://purl.obolibrary.org/obo/PO_0007026	E-MTAB-8628	ERR3773822	Anja Fullgrabe	13/01/21 18:40
+developmental stage	third instar larvae stage	http://www.ebi.ac.uk/efo/EFO_0002682	E-GEOD-62008	GSM1518370	Anja Fullgrabe	13/01/21 18:40
+diet	high-fat diet	http://www.ebi.ac.uk/efo/EFO_0002757	E-MTAB-7755	HFD_104:biotin	Anja Fullgrabe	13/01/21 18:40
+disease	Adenomyosis	http://www.ebi.ac.uk/efo/EFO_1001757	E-GEOD-7307	GSM176257	Anja Fullgrabe	13/01/21 18:40
+disease	Anaplastic (Malignant) Meningioma	http://www.ebi.ac.uk/efo/EFO_1000082	E-MTAB-4999	ctrl5	Anja Fullgrabe	13/01/21 18:40
+disease	Biliary atresia	http://www.orpha.net/ORDO/Orphanet_30391	E-GEOD-15235	GSM380382	Anja Fullgrabe	13/01/21 18:40
+disease	Corticobasal degeneration	http://www.orpha.net/ORDO/Orphanet_278	E-PROT-58	ppb.iBAQ.UniPenn_Corticobasal.degeneration.2	Anja Fullgrabe	13/01/21 18:40
+disease	Genetic frontotemporal degeneration with dementia	http://www.orpha.net/ORDO/Orphanet_276061	E-PROT-58	ppb.iBAQ.UniPenn_Frontotemporal.dementia.with.parkinsonism.linked.to.chromosome.17_2	Anja Fullgrabe	13/01/21 18:40
```

Comment:
for disease terms did we decide on an ontology? I thought we weren't using Orphanet as a source in general

Reply:
This term is also in EFO.